osu.Game/Screens/Edit/Compose/Components/Timeline/TimelineHitObjectBlueprint.cs

```diff
@@ -340,22 +359,38 @@ protected override void OnDragEnd(DragEndEvent e)
             }
         }
 
+        public class Border : ExtendableCircle
+        {
+            [BackgroundDependencyLoader]
+            private void load(OsuColour colours)
+            {
+                Content.Child.Alpha = 0;
+                Content.Child.AlwaysPresent = true;
+
+                Content.BorderColour = colours.Yellow;
+                Content.EdgeEffect = new EdgeEffectParameters();
```

Comment:
Is this doing anything?

Reply:
Had to remove the edge effect applied below, else it shows up in the empty space and makes everything dark. I can split out the base class if you think it's worth it, and only apply the `EdgeEffect` in the other implementation.

Reply:
Ah, I somehow missed that there was an edge effect there in my original review :open_mouth: